Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1

```diff
@@ -234,11 +234,19 @@ function Main {
             -RunInBackground -runAsSudo
 
         Write-LogInfo "Monitoring test run..."
+        $ForceExitCount = 0
+        $MaxCount = 10
         while ((Get-Job -Id $testJob).State -eq "Running") {
             $currentStatus = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
                 -username "root" -password $password -command "tail -1 fioConsoleLogs.txt" `
                 -runAsSudo
             Write-LogInfo "Current Test Status: $currentStatus"
+            if ($currentStatus -imatch "Doing forceful exit of this job") {
+                $ForceExitCount++
+                if ( $ForceExitCount -eq $MaxCount) {
+                    throw "FIO is stuck, exiting the test"
```

Comment:
let's change 'exiting the test' -> 'aborting the test.'

Reply:
modified exception log